test/hotspot/jtreg/runtime/CommandLine/VMOptionWarning.java

```diff
@@ -82,6 +93,13 @@ public static void main(String[] args) throws Exception {
                 output.shouldContain("Error: VM option 'PrintInlining' is diagnostic and must be enabled via -XX:+UnlockDiagnosticVMOptions.");
                 break;
             }
+            case "DiagnosticCompileCommand": {
+                pb = ProcessTools.createLimitedTestJavaProcessBuilder("-XX:CompileCommand=PrintAssembly,*::*", "-version");
```

Comment:
We don't really need to print assembly here, right? It would just unnecessarily slow down test execution. I would suggest to use something like `-XX:CompileCommand=PrintAssembly,MyClass::myMethod`

Reply:
No indeed, that's a good point. Replaced.